concepts/arithmetic-operators/introduction.md

```diff
@@ -0,0 +1,35 @@
+# Introduction to Arithmetic Operators
```

Comment:
@ErikSchierboom Is it a problem if the h1 header is more than just the word "Introduction"? (same question for about)

Reply:
NOTE: if this matters, this is not the only one that would have a problem. Also, it is not mentioned in the contributing guides so it really must be in there if this is a thing that matters.

Reply:
@junedev @jmrunkle It doesn't matter. We strip the h1 heading